app/src/test/java/org/mozilla/fenix/perf/StartupReportFullyDrawnTest.kt

```diff
@@ -37,9 +43,10 @@ class StartupReportFullyDrawnTest {
     @Before
     fun setup() {
         MockKAnnotations.init(this)
+        val binding = TopSiteItemBinding.inflate(LayoutInflater.from(testContext), rootContainer, false)
```

Comment:
Inflate the needed layout to avoid the 
```
java.lang.NullPointerException: Missing required view with ID: 
	at org.mozilla.fenix.databinding.TopSiteItemBinding.bind(TopSiteItemBinding.java:97)
```
errors